### ISSUE TITLE
deadcode in eventloops.py

### ISSUE THREAD

**Issue by Carreau** [ipython/ipykernel#349]
```
    from IPython.external.qt_for_kernel import QtCore

    if _use_appnope() and kernel._darwin_app_nap:
        from appnope import nope_scope as context
    else:
        from contextlib import contextmanager
        @contextmanager
        def context():
            yield
```

`context` seem to be unused in rest of the file.

**Comment by matholiveira91** [comment]
#import pltaform
kernel=plantaform.uname(2)
if _use appnope() and kernel:

**Comment by matholiveira91** [comment]
#try to make it generic

**Comment by osule** [comment]
@matholiveira91 I don't see `context` used outside this module in the project. 
Do you supposed that it is used in some other project?

I can help on this issue.
cc: @Carreau

**Comment by Carreau** [comment]
closed by #355 ans #357 Good job all.
